imagepipeline-base/src/main/java/com/facebook/cache/disk/DefaultDiskStorage.java

```diff
@@ -107,6 +116,25 @@ public DefaultDiskStorage(
     mClock = SystemClock.get();
   }
 
+  private static boolean isExternal(File directory) {
+
+    boolean state = false;
+    String cacheDirPath = null;
+    File extStoragePath = Environment.getExternalStorageDirectory();
+    if(extStoragePath != null) {
+      cacheDirPath = extStoragePath.toString();
+      try {
+        String appCacheDirPath = directory.getCanonicalPath();
+        if (appCacheDirPath.contains(cacheDirPath)) state = true;
```

Comment:
Nit: coding style, we always surround code blocks with `{ }` even if it's a single statement.